mockserver.js

```diff
@@ -88,25 +88,41 @@ function getBodyOrQueryString(body, query) {
  * fancy body parsing in mockserver
  * for now).
  */
-function getBody(req) {
+function getBody(req, callback) {
   var body = '';
   
   req.on('data', function(b){
-    body = body + b.toString()
+    body = body + b.toString();
   });
-  
-  return body;
```

Comment:
in this way body was always `''` since this function returns as soon as it's called without waiting for the data collection to be over